presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -101,6 +103,46 @@ public void testExecuteQuery()
         }
     }
 
+    @Test
+    public void testGetMetadata()
+            throws Exception
+    {
+        try (Connection connection = createConnection("blackhole", "blackhole")) {
+            try (Statement statement = connection.createStatement()) {
+                statement.execute("CREATE TABLE test_get_metadata (" +
+                        "c_boolean boolean, " +
+                        "c_decimal decimal, " +
+                        "c_decimal_2 decimal(10,3)," +
+                        "c_varchar varchar, " +
+                        "c_varchar_2 varchar(10), " +
+                        "c_row row(x integer, y array(integer)), " +
+                        "c_array array(integer), " +
+                        "c_map map(integer, integer))");
+            }
+
+            List<String> colNames = Arrays.asList("c_boolean", "c_decimal", "c_decimal_2",
+                    "c_varchar", "c_varchar_2", "c_row", "c_array", "c_map");
+            List<String> expectColTypes = Arrays.asList("boolean", "decimal(38,0)", "decimal(10,3)",
+                    String.format("varchar(%s)", Integer.MAX_VALUE), "varchar(10)", "row()", "array", "map");
```

Comment:
`String.` is redundant. We are using static import and omit the class name when the method name is self descriptive. `format` method is one of the example. 

Also, I think writing literal assertions as below is better than loop with list. When we want to add other assertions later (eg `getColumnType`), adding new list makes readability low. What do you think about it? 
```java
                assertEquals(metadata.getColumnCount(), 8);
                for (int i = 1; i <= metadata.getColumnCount(); i++) {
                    assertEquals(metadata.getCatalogName(i), "blackhole");
                    assertEquals(metadata.getSchemaName(i), "blackhole");
                    assertEquals(metadata.getTableName(i), "test_get_metadata");
                }

                assertEquals(metadata.getColumnName(1), "c_boolean");
                assertEquals(metadata.getColumnTypeName(1), "boolean");

                assertEquals(metadata.getColumnName(2), "c_decimal");
                assertEquals(metadata.getColumnTypeName(2), "decimal(38,0)");
...
```

Reply:
@ebyhr thank you so much! I agree!